docs/1.0. EDID-Mapping.md

```diff
@@ -0,0 +1,195 @@
+# AMWA BCP-XXX-YY: EDID Mapping in NMOS Systems \[Work In Progress\]
+{:.no_toc}
+
+* A markdown unordered list which will be replaced with the ToC, excluding the "Contents header" from above
+{:toc}
+
+## Background
+
+AMWA NMOS specifications provide resources for specifying receiver capabilities of an NMOS device.  Recent extension of these resources allow NMOS Controllers developed for PRO A/V User-Stories to be created.  In particular, these NMOS Controllers include Controllers developed for the AIMS sponsered IPMX set of open standards [IPMX]. 
+
+
+## Scope
+
+This document provides Best Current Practice guideluines for how to implement mappings of EDID elements onto AMWA IS-04 and IS-EDID-CM resources.  The document also provides guidance on how implementers may use the APIs to achieve some common IPMX type User-Stories and associated use-cases as described in the EDID Connection Management Architecture [EDID-CM-ARCH].
+
+
+## Definitions
+
+
+_See also the [NMOS Glossary](https://github.com/AMWA-TV/nmos/wiki/Glossary), and definitions within RFCs._
+
+### EDID 
+> Extended Display Identification Data (EDID) is a metadata format for display devices to describe their capabilities to a video source (e.g. graphics card or set-top box). The data format is defined by a standard published by the Video Electronics Standards Association (VESA).  For purposes of this BCP, EDID definitions are taken from the E-EDID version 1.4 [E-EDID].
+
+
+
+## Mapping from EDID to NMOS Resources
+
+This section is broken down into several subsections that describe recommendations for mapping parts of a target display device's EDID to NMOS Resources.  The following subsections are provided:
+
+> - Mapping of EDID to IS-04 Receiver Caps 
+>> - Video Mappings from EDID to IS-04 Receiver Caps
+>> - Audio Mappings from EDID to IS-04 Receiver Caps
+> - Mapping of EDID to IS-EDID-CM
+>> - EDID to IS-EDID-CM Destination Resource
+>> - EDID to Essential Media Capabilities
+>> - EDID to Raw EDID Format
```

Comment:
I see you used quotes for the lists, if you'd like you can use indentation with markdown

```suggestion
- Mapping of EDID to IS-04 Receiver Caps 
   - Video Mappings from EDID to IS-04 Receiver Caps
   - Audio Mappings from EDID to IS-04 Receiver Caps
- Mapping of EDID to IS-EDID-CM
   - EDID to IS-EDID-CM Destination Resource
   - EDID to Essential Media Capabilities
   - EDID to Raw EDID Format
```

Reply:
Thanks Chris - will do.